docs/development.md

```diff
@@ -382,8 +439,30 @@ The `lang` property of the `<script>` block is used to specify the used programm
  * for Javascript, use `<script lang="javascript"> ... </script>`
  * for Python, use `<script lang="python"> ... </script>`
 
-`<script>` also supports `tags`. For more information, see the dedicated section **Plugins and tags**.
+`<script>` also supports `tags`. For more information, see the dedicated section for [`tags`](development?id=tags).
+
+## Use custom libraries
+You can specify plugin requirements of ImJoy in the dedicated field in
+its `config` [block](http://localhost:8000/docs#/development?id=requirements)
+
+This allows you include public libraries with `pip` or `conda`, but also your own
+libraries. Below we describe some of the most commonly encountered scenarios.
```

Comment:
I think the following options (except the last one) should be placed in the section of `requirements`, because it does not limited to custom libraries, but any libraries.

For custom libraries, you can mention how to construct the pip url `pip:git+https://...` for requirements.